app/code/Academy/ProductFeed/Ui/Component/ProductFeed/Form/Categories/ProductFeedCategoriesTreeOptions.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Academy\ProductFeed\Ui\Component\ProductFeed\Form\Categories;
+
+use Magento\Catalog\Ui\Component\Product\Form\Categories\Options;
+
+class ProductFeedCategoriesTreeOptions extends Options
```

Comment:
Czy ta klasa jest potrzebna? IMO można podać w `product_feed_manage_form.xml` tą klasę, którą rozszerzasz.

Reply:
Myślałam o tym, że może da się to category tree jakoś fajnie rozszerzyć żeby lepiej wyglądało. Ale może na ten moment faktycznie lepiej to poprawić a jak będzie potrzeba to się to wtedy zrobi.